configs/neox_arguments.md

```diff
@@ -1766,16 +1774,6 @@ Args for deepspeed config
 
 
 
-- **bf16**: dict
-
-    Default = None
-
-    Configuration for using bfloat16 floating-point format as an alternative to FP16. BFLOAT16 requires hardware support (e.g., NVIDIA A100).
```

Comment:
Why'd we remove these details?

Reply:
They're tied to the specific `bf16` config that you wanted removed. I can move them to the section on the `precision` config?